cl/users/tasks.py

```diff
@@ -102,3 +106,35 @@ def update_mailchimp(self, email, status):
             email,
             r.status_code,
         )
+
+
+@app.task(bind=True, max_retries=3, interval_start=5 * 60)
+def process_retry_email(
+    self: Task,
+    failed_pk: int,
+) -> int:
+    """Task to retry failed email messages"""
+
+    failed_email = FailedEmail.objects.get(pk=failed_pk)
+    if failed_email.status != STATUS_TYPES.SUCCESSFUL:
+        # Only execute this task if it has not been previously processed.
+        failed_email.status = STATUS_TYPES.IN_PROGRESS
+        failed_email.save()
+        # Compose email from stored message.
+        email = failed_email.stored_email.convert_to_email_multipart()
+        try:
+            email.send()
+        except Exception as exc:
```

Comment:
Let's try to get the exceptions correct here. I fear catching all exceptions like this. It's hidden some nasty bugs in the past. Even if we can't guess all the exceptions at first, we'll catch them eventually.

Reply:
Sure, I added, for now, the following exceptions:
```
botocore_exception.HTTPClientError
botocore_exception.ConnectionError
```
That I could catch locally after losing network connection.